File: div_by_4_or_6_not_both.cpp
// Print numbers from 1 to 100 divisible by 4 or 6 but not both
// Print 5 numbers per line
// Show count and sum

#include <iostream>
using namespace std;

int main() {
    int count = 0, sum = 0;

    for (int i = 1; i <= 100; i++) {
        if ((i % 4 == 0 || i % 6 == 0) && i % 12 != 0) {
            cout << i << " ";
            count++;
            sum += i;
            if (count % 5 == 0) cout << endl;
        }
    }

    cout << "\nCount: " << count << endl;
    cout << "Sum: " << sum;
    cout<< endl;

    return 0;
}

File: div_by_4_or_6_not_both.py
# Print numbers from 1 to 100 divisible by 4 or 6 but not both
# Print 5 numbers per line
# Show count and sum

count = 0
sum = 0

for i in range(1, 101):
    if (i % 4 == 0 or i % 6 == 0) and i % 12 != 0:
        print(i, end=" ")
        count += 1
        sum += i

        # print 5 numbers per line
        if count % 5 == 0:
            print()

print("\nCount:", count)
print("Sum:", sum)
